lib/AST/Module.cpp

```diff
@@ -592,10 +592,6 @@ ModuleDecl::lookupConformance(Type type, ProtocolDecl *protocol) {
   // existential's list of conformances and the existential conforms to
   // itself.
   if (type->isExistentialType()) {
-    // FIXME: Recursion break.
```

Comment:
Are you sure this is not needed? existentialConformsToSelf() can call back into validateDecl() IIRC

Reply:
No longer! It uses simpler queries that don't go through `validateDecl()`, now that `getInheritedProtocols()` is all decl-lookup-based.